main/src/ca/uwaterloo/flix/api/Flix.scala

```diff
@@ -217,7 +217,7 @@ class Flix {
       case tast =>
         val ast = PropertyGen.collectProperties(tast)
         val sast = Simplifier.simplify(ast)
-        val lifted = LambdaLift.lift(sast)
+        val lifted = Tailrec.tailrec(LambdaLift.lift(sast))
```

Comment:
An open-ended comment: as we add more phases, this will get messier. Is there anything better we can do?